css/main.css

```diff
@@ -1,3 +1,248 @@
-/* Add your custom CSS here */
+.title-row {
+  justify-content: center;
+}
+.title-row p {
+  color: white;
+  font-size: 2rem;
+}
+.main-centre {
+  margin-top: 30px;
+  font-size: 1em;
+}
+.main-centre p {
+  text-align: center;
+}
+.about-section {
+  margin: 15px;
+}
+.row2 {
+  display: flex;
+  flex-wrap: wrap;
+  margin-right: 0px;
+  margin-left: 0px;
+}
+.about-col-6 {
+  position: relative;
+  min-height: 1px;
+  padding-right: 0px;
+  padding-left: 0px;
+}
+.about-col-6 h2 {
+  font-size: 80px;
+}
+.aboutus-text {
+  margin: 5rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+body {
+  font-size: 20px;
+}
+.content {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+.aboutus-text p {
+  color: white;
+}
+.about {
+  background-color: rgba(0, 0, 0, 0.3);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 1.3em;
+  text-align: center;
+  font-size: 1.3em;
+  text-align: center;
+}
+.image-about {
+  padding: 0;
+}
+.about h1 {
+  text-align: center;
+  color: red;
+}
 
-/* Dont' forget to link this file to your HTML in the <head> */
+h5 {
+  text-align: center;
+  color: rgb(26, 25, 24);
+}
+h2 {
+  text-align: center;
+  color: rgb(138, 25, 25);
+}
+h1 {
+  color: white;
+  font-size: 3rem;
+}
+img {
+  width: 100%;
+}
+h3 {
+  color: rgb(138, 25, 25);
+  font-size: 2rem;
+}
+h4 {
+  text-align: center;
+  font-weight: bold;
+}
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  margin-right: -15px;
+  margin-left: -15px;
```

Comment:
Why write this yourself? That's what grid.css is for 🙂 It's all in there already!